src/main/java/org/sagebionetworks/web/client/PortalGinModule.java

```diff
@@ -1455,7 +1459,8 @@ protected void configure() {
         bind(SynapseContextPropsProvider.class).to(SynapseContextPropsProviderImpl.class);
         bind(AddToDownloadListV2.class).to(AddToDownloadListV2Impl.class);
         bind(OpenDataView.class).to(OpenDataViewImpl.class);
-
         bind(QueryClientProvider.class).to(QueryClientProviderImpl.class).in(Singleton.class);
+        bind(IntendedDataUseReportWidgetView.class).to(IntendedDataUseReportWidgetViewImpl.class);

```

Comment:
Added an IntendedDataUseReport view implementation (that utilizes the SRC component)